setup.py

```diff
@@ -5,14 +5,15 @@
 import setuptools
 
 
-PACKAGENAME = 'sqre-github-snapshots'
+PACKAGENAME = 'sqre-github-snapshot'
 DESCRIPTION = 'LSST Data Management SQuaRE Github Organization Snapshotter'
 AUTHOR = 'Adam Thornton'
 AUTHOR_EMAIL = 'athornton@lsst.org'
 URL = 'https://github.com/lsst-sqre/sqre-github-snapshots/'
 VERSION = '0.1.0'
 LICENSE = 'MIT'
 
+
 def read(filename):
     '''Convenience function to do, basically, an include'''
```

Comment:
Just a minor FYI that we standardize no three double-quotes rather than three single quotes for our docstrings.